features/current_iterations.feature

```diff
@@ -0,0 +1,37 @@
+Feature: See all customers who haven't given feedback 
+	As a logged in user
+	So that I can easily see which customers to remind to fill out a feedback form
+    I want to see all iterations that don't have feedback yet
```

Comment:
SMART definition of goal! If you end up committing again, you could change this line's identation to match the other lines. Apart from that, very good test!